diff --git a/src/emacs.c b/src/emacs.c
index eddd729..ad3f034 100644
--- a/src/emacs.c
+++ b/src/emacs.c
@@ -2166,9 +2166,6 @@ You must run Emacs in batch mode in order to dump it.  */)
   if (! noninteractive)
     error ("Dumping Emacs works only in batch mode");
 
-  if (!might_dump)
-    error ("Emacs can be dumped only once");
-
 #if defined GNU_LINUX && !defined CANNOT_DUMP
 
   /* Warn if the gap between BSS end and heap start is larger than this.  */
